index.html

```diff
@@ -30,68 +30,72 @@
+            <!-- The logo & description -->
+            <div class="organize-landing-page">
+                <figure>
+                    <a href="https://zenplayer.audio/" title="Zen Audio Player" class="zen-logo">
+                        <img src="img/zen-audio-player-905.png" class="img-100" alt="Zen Audio Player logo">
+                    </a>
+                </figure>
 
-        <hgroup>
-            <h1>Zen Audio Player</h1>
-            <h2 class="color-grey">Listen to YouTube videos, without the distracting visuals.</h2>
-            <h2 class="color-grey hide flash" id="mobile-message"></h2>
-        </hgroup>
-    </header>
-    <div id="container">
-        <!-- The form -->
-        <form id="form">
-            <div class="input-group">
-                <input type="text" id="v" name="v" class="input-search color-grey" placeholder="Search, YouTube video ID, or URL">
-                <span class="input-group-button">
-                    <button class="btn btn-search" id="submit">
-                        <i class="fa fa-search"></i>
-                    </button>
-                </span>
+                <hgroup>
+                    <h1>Zen Audio Player</h1>
+                    <h2 class="color-white">Listen to YouTube videos, without the distracting visuals.</h2>
+                    <h2 class="color-grey hide flash" id="mobile-message"></h2>
+                </hgroup>
             </div>
-        </form>
-        <div id="search-results">
-            <h3>Search Results</h3>
-            <ul></ul>
-        </div>
-        <div id="zen-error" class="hide flash flash-error"></div>
-        <!-- The video, along with related content & messages -->
-        <div id="audioplayer">
-            <h3>
-                <a id="zen-video-title" class="plyr__cite plyr__cite--video"></a>
-            </h3>
-
-            <div id="playerControls">
-                <button class="btn" id="togglePlayer">
-                    Show Player
-                </button>
-                <button class="btn" id="toggleDescription">
-                    Show Description
-                </button>
-                <input id="toggleRepeat" name="toggleRepeat" type="checkbox" />
-                <label id="toggleRepeatLabel" for="toggleRepeat">Repeat track</label>
+        </header>
+        <div id="container">
+            <!-- The form -->
+            <form id="form">
+                <div class="input-group">
+                    <input type="text" id="v" name="v" class="input-search color-grey" placeholder="Search, YouTube video ID, or URL">
+                    <span class="input-group-button">
+                        <button class="btn btn-search" id="submit">
+                            <i class="fa fa-search"></i>
+                        </button>
+                    </span>
+                </div>
+            </form>
+            <div id="search-results">
+                <h3>Search Results</h3>
+                <ul></ul>
```

Comment:
The search results now break the layout:

![search-results](https://user-images.githubusercontent.com/1480063/31479119-f494309e-aec9-11e7-8e34-763e0eb4272e.png)